spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -139,6 +139,20 @@
     end
   end
 
+  describe "#update_content" do
+    it 'calls ingest_file' do
+      expect_any_instance_of(Hyrax::Actors::FileActor).to receive(:ingest_file).with(local_file, true)
+      actor.update_content(local_file)
+    end
+    it 'runs callbacks' do
+      expect(Hyrax.config.callback).to receive(:run).with(:after_update_content, file_set, user)
+      actor.update_content(local_file)
+    end
+    it "returns true" do
+      expect(actor.update_content(local_file)).to be true
```

Comment:
Please stub out the update so this doesn't have to persist anything. This will keep the test suite from getting too slow.